llvm-bazel/configure.bzl

```diff
@@ -10,4 +10,4 @@ load(":overlay_directories.bzl", "overlay_directories")
 OVERLAY_DIR = "llvm-project-overlay"
 
 def llvm_configure(name, overlay_path = OVERLAY_DIR, **kwargs):
-    overlay_directories(name=name, overlay_path=overlay_path, **kwargs)
+    overlay_directories(name = name, overlay_path = overlay_path, **kwargs)
```

Comment:
I've seen this and some other minor Bazel fixes in a two PRs -- maybe factor it out as just cleanup?

Reply:
Done in #50